test/vanilla/__snapshots__/ignite-remove-demo.test.ts.snap

```diff
@@ -6,6 +6,7 @@ exports[`ignite-cli remove-demo should print the expected response 1`] = `
 removing file /user/home/ignite/.maestro/flows/FavoritePodcast.yaml
 removing file /user/home/ignite/.maestro/flows/Login.yaml
 removing file /user/home/ignite/.maestro/shared/_Login.yaml
+removing file /user/home/ignite/.maestro/shared/_OnFlowStart.yaml
```

Comment:
Will need to update this snap if we decide to keep this file with my comment on it above